build.gradle

```diff
@@ -36,9 +41,9 @@ buildscript {
         jcip_annotation_version = "1.0"
 
         // Testing
-        junit_version = "4.12"
+        junit_version = "4.13.2"
```

Comment:
chore: remove or provide this parameter for junit-jupiter

Reply:
Okay, I've added a `jupiter_version`.

I added a new parameter instead of replacing is one because `junit_version` still appears in the android packages. I was afraid to mess with those. Should I convert the android packages to JUnit 5 too? Do those tests run in my normal IDE, or do I have to switch to Android Studio for that?